docs/.vuepress/components/ApiProp.vue

```diff
@@ -34,7 +34,7 @@
         <cdr-col
           span="12"
         >
-          <p aria-lable="prop description" class="prop-description">
+          <p aria-label="prop description" class="prop-description">
```

Comment:
as these are not table cells there is nothing preventing the label from floating into the space reserved for the value - I assume that this will need to stay below that region to ensure an expected user experience